js-advanced-unit-5/vending-machines/cart-functions.js

```diff
@@ -0,0 +1,59 @@
+const calculateChange = (payment, total) => {
+  const change = Math.round((payment - total) * 100) / 100
+  return change
+}
+
+const isSufficientPayment = (payment, total) => {
+  if (Math.round((payment - total) * 100) / 100 >= 0) {
+    return true
+  } else {
+    return false
+  }
```

Comment:
Since the conditional will evaluate to a boolean, this can also be re-written as:

```suggestion
  return Math.round((payment - total) * 100) / 100 >= 0
```